lib/storage.js

```diff
@@ -38,6 +38,46 @@ const path = require('path');
 const utils = require('./utils');
 
 /**
+ * @summary Set data path
+ * @function
+ * @public
+ *
+ * @description
+ * Pass the directory you want to call the functions from.
+ * If no directory is passed, returns to default 'storage'
+ *
+ * @param {String} directory - directory
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.setDataPath('foo');
+ */
+exports.setDataPath = function(dir = ""){
```

Comment:
Now that we're exposing both functions from `utils`, what do you think about simply moving their logic here?